CharacterCreation/02_Classes.txt

```diff
@@ -143,8 +143,11 @@ Skill Proficiencies:
 
 
 =====   DOCTOR   =====
-You know how or why something works [most likely humans / humanoid bodies), 
-and put that knowledge to good use by supporting your team.
+
+    You know the secrets of life and death. The very continued existance of 
+those around you depend on you striking the right balance between heal and harm. 
```

Comment:
gr:"depends"
Also, I'd go with "on whether you strike the right balance between heal and harm."